usaspending_api/awards/v1/views.py

```diff
@@ -182,3 +185,57 @@ def get_queryset(self):
         filtered_queryset = self.filter_records(self.request, queryset=queryset)
         ordered_queryset = self.order_records(self.request, queryset=filtered_queryset)
         return ordered_queryset
+
+
+class FABAFilter(filters.FilterSet):
+
+    class Meta:
+        model = FinancialAccountsByAwards
+        # You can create any relationships you want here using ORM notation
+        fields = ('piid', 'treasury_account__agency_id')
+
+
+class FinancialAccountsByAwardsListViewset(viewsets.ReadOnlyModelViewSet):
+
+    '''
+        I spent a lot of time investigating the serializers here trying to come up with a better solution. However
+        ultimately I determined that the LimitableSerializer class which we base all our serializers on is actually
+        fairly well put together. It handles prefetching well, so that the nested serializers don't prompt new queries,
+        and it doesn't really mess with the core functionality of the serializer as written in DRF. My main objection
+        to them is having to specify default_fields unless verbose is set. If I was ultimately going to rewrite this,
+        I might switch that around so that verbose is the default. I did not feel that that was worth investing time in
+        for this investigation however.
+    '''
+
+    serializer_class = FinancialAccountsByAwardsSerializer
```

Comment:
Where is the class definition of this Serializer in this PR?

Seeing that is part of AC 2 and 4
> 2. A developer can successfully use serializers adapting a model object in DRF
> 4. Not all fields of the model are returned to the API user (they can be limited)

It'd be good to see how you manage only allowing explicit declared fields to be returned.